docs/architecture/RECON-CORE-BOUNDARY.md

```
@@ -0,0 +1,100 @@
+BreachEvaluator(threshold=Decimal("0.01"), breach_kind="BREAK")
+
+# CASS 7.15 (services/recon/recon_agent.py)
+BreachEvaluator(threshold=BREACH_HITL_THRESHOLD, breach_kind="HITL")  # Decimal("100")
+```
+
+## Shared boundary rule (equivalence-critical)
+
+Both regimes previously used **strict `>`** for breach and **`<=`** for clear/match. The core
+preserves this exactly:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)